libs/perun/components/src/lib/groups-list/groups-list.component.html

```diff
@@ -15,15 +18,18 @@
             </mat-checkbox>
           </th>
           <td *matCellDef="let row" class="static-column-size" mat-cell>
-            <mat-checkbox (change)="$event ? itemSelectionToggle(row) : null"
-                          (click)="$event.stopPropagation()"
-                          [aria-label]="checkboxLabel(row)"
-                          [checked]="selection.isSelected(row)"
-                          [disabled]="(row.name === 'members' && disableMembers) || disableSelect(row.id)"
-                          matTooltip="{{'SHARED_LIB.PERUN.COMPONENTS.GROUPS_LIST.DISABLED_HINT' | translate}}"
-                          [matTooltipDisabled]="!disableSelect(row.id)"
-                          color="primary">
-            </mat-checkbox>
+            <span
+              [matTooltip]="(authType === 'create-relation-dialog' ? 'SHARED_LIB.PERUN.COMPONENTS.GROUPS_LIST.CREATE_RELATION_AUTH_TOOLTIP' :
+               'SHARED_LIB.PERUN.COMPONENTS.GROUPS_LIST.ALREADY_MEMBER_TOOLTIP') | translate"
```

Comment:
better code readability - translate in in the ts file and here you get only variables